docs/source/conf.py

```diff
@@ -175,3 +180,22 @@ def setup(app):
     app.add_config_value('m2r_disable_inline_math', False, 'env')
     app.add_directive('mdinclude', MdInclude)
     app.add_stylesheet('custom.css')
+
+    with open(
+        os.path.join(os.path.dirname(os.path.realpath(__file__)), 'component_docs.rst'),
+        'w',
+    ) as f:
+        f.write("labscript suite components\n")
+        f.write("==========================\n")
+        f.write("    \n")
+        f.write(".. toctree::\n")
+        f.write("    :maxdepth: 2\n")
+        f.write("    \n")
+        if project != "the labscript suite":
+            f.write(
+                "    labscript suite (metapackage)<{}>\n".format(
+                    intersphinx_mapping['labscript-suite'][0]
+                )
+            )
+        for ls_prog in labscript_suite_programs:
+            f.write("    {} <{}>\n".format(ls_prog, intersphinx_mapping[ls_prog][0]))
```

Comment:
The intersphinx mapping from an earlier PR has an errant underscore in `docs.labscript_suite.org`.

Reply:
Done